intallation.md

```diff
@@ -0,0 +1,101 @@
+# Index
```

Comment:
Besides the typo, how about we call this `INSTALL.md`, so that it would be consistent with the `README.md` and more easily discoverable in the repository's root?

Reply:
`Index` it's like index in a book. Didn't mean `Install`. 😅